### PR TITLE
chore: update go sdk document

### PR DESCRIPTION
**Pull request by xuthus5**

### CODE REVIEW

**Comment by xuthus5** [comment]
@xiangyu5632 @shoothzj PTAL.

**Comment by shoothzj** [comment]
@xiangyu5632 I think it's hard to maintain docs in two side. For more detail or advance usage, can we link to client's README directly?

File: src/zh/guide/develop/client_go.md [hunk 1]
```diff
@@ -120,3 +114,76 @@ for _, r := range res.Results {
...
+cmd := fmt.Sprintf("show field keys")
+value, err := client.ShowFieldKeys("db0", cmd)
+if err != nil {
+	fmt.Println(err)
+}
+for _, result := range values {
+	fmt.Printf("measurement: %s\n", result.Measurement) // cpu
+	fmt.Printf("kv: %+v\n", result.Values) // [{Name:value Value:float}]
+}
+```
```
Comment (shoothzj) [inline src/zh/guide/develop/client_go.md:189]: could you please add a empty new line

**Comment by xuthus5** [comment]
For minimal changes, I resubmitted a PR, which was closed

**Pull request closed without merging**